UEBPrep/test/test_helper.rb

```diff
@@ -1,3 +1,4 @@
+ActiveRecord::Migration.maintain_test_schema!
```

Comment:
Make sure that our test database stays in sync with the structure of our development database.